src/playbacks/no_flash/public/style.scss

```diff
@@ -0,0 +1,25 @@
+[data-no-flash] {
+  z-index: 1000;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+}
+
+[data-no-flash] p[data-no-flash-msg] {
+  position: absolute;
+  font-size: 25px;
+  top: 40%;
+  color: white;
+}
+
+[data-no-flash] p[data-no-flash-msg] a {
```

Comment:
Merge rule `[data-no-flash] p[data-no-flash-msg] a` with rule on line 1<br>Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3